src/public/index.styl

```diff
@@ -8,6 +8,17 @@
     width 100%
     top-padded(30px)
 
+@media (max-width: (768/basefont)rem)
+    .pho-public-layout--full
```

Comment:
It didn't but since this PR is about the errorShare view and this fix an issue about errorShare i put it there.